.gitmodules

```diff
@@ -19,3 +19,6 @@
 [submodule "libraries/3rdparty/CMock"]
 	path = libraries/3rdparty/CMock
 	url = https://github.com/ThrowTheSwitch/CMock.git
+[submodule "vendors/espressif/esp-idf"]
+	path = vendors/espressif/esp-idf
+	url = https://github.com/espressif/esp-afr-sdk.git
```

Comment:
Noticed that the repo https://github.com/espressif/esp-afr-sdk.git, doesn't have a master branch. Is the plan not to have a master branch in the future as well? I think it will be better to specifically mention the branch in .gitmodules, if there is a remote possibility of creating a master branch.

Reply:
@leegeth Branching name is based on underlying ESP-IDF release version. Yes, do not see that there will `master` branch in this repository (mostly base software will based on some stable `release` branch). Have update branch, please have a look.

Reply:
Thanks. Looks good